src/main/resources/anniv5Resources/localization/eng/cosmoscommandpack/Powerstrings.json

```diff
@@ -10,7 +10,10 @@
   "${ModID}:RecallPower": {
     "NAME": "Recall",
     "DESCRIPTIONS": [
-      "Whenever a card is #yAmplified, put a card from your #yExhaust pile into your hand."
+      "The first time a card is #yAmplified each turn, put ",
```

Comment:
Minor, but with how this is set up, the number here won't be highlighted (when the power amount is >1). You'll need to make a separate entry with a #b for the >1 case (since the =1 case doesn't need highlighting).